ui/app/ducks/gas/gas.duck.js

```diff
@@ -175,28 +143,10 @@ export function gasEstimatesLoadingFinished() {
 }
 
 async function queryEthGasStationBasic() {
```

Comment:
This method should be renamed. We should check for other references to "ethgasstation" as well.

Reply:
Done in 1b9ffa517